src/Components/NavBar.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import logo from "../images/logo.png";
+
```

Comment:
Please take a look at this extra spacing between imports, you can keep it consistent throughout all files.

Reply:
Yes I have removed extra spaces from where ever required.